tests/specs/graph/jsr/DynamicImport.txt

```diff
@@ -0,0 +1,133 @@
+# https://jsr.io/@scope/a/meta.json
+{"versions": { "1.0.0": {} } }
+
+# https://jsr.io/@scope/a/1.0.0_meta.json
+{
+  "exports": {
+    ".": "./mod.ts"
+  }
+}
+# https://jsr.io/@scope/a/1.0.0/mod.ts
+const test: typeof import('./a.ts') = await import("./a.ts");
```

Comment:
I'm going to do a separate PR to not require an explicit type here for fast check.

Reply:
It's actually significantly more owrk to implement this than I expected, so I'm going to bail on it for now.